assets/chat/js/const.js

```diff
@@ -48,4 +48,11 @@ function isKeyCode(e, code){
     return getKeyCode(e) === code
 }
 
-export {KEYCODES,DATE_FORMATS,isKeyCode,getKeyCode}
\ No newline at end of file
+// emote:key; css class to apply
+const GENERIFY_OPTIONS = {
```

Comment:
Why is this an object and not an array? It looks like you're just using the keys, right?

Reply:
The values are the CSS classes to apply.

Reply:
Didn't see that file for some reason